tests/test_validation_rules.py

```diff
@@ -182,3 +184,42 @@ def test_enum_value_removing_desc():
     ''')
     diff = Schema(a, b).diff()
     assert RemoveEnumValueDescription(diff[0]).is_valid() is False
+
+
+def test_schema_added_field_no_desc():
```

Comment:
Can we add a test for double nested fields? Something Like `Type.Field.SubField`?